app/src/main/res/values-sw360dp-v13/preference.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--Created By Me to fix the appearance issue caused by migrating to AndroidX-->
```

Comment:
We shouldn't need a comment for this -- the git history can add this context 🙂

Reply:
Nvm about that comment. i have changes so many files till now. I have put there everywhere in my local copy just incase if anything breaks, I know exactly where to look at. 😄